### ISSUE TITLE
When deleting a lot of mails from the inbox, make sure older ones on the bottom are loaded automatically

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/mail#1595]
This happens a lot when removing notifications from Twitter for example. The list suddenly ends. Instead of that, the »Load more« action should automatically be carried out at some point.

**Comment by ChristophWurst** [comment]
right. we could trigger the load-more-action if a certain threshole was reached, e.g. 15msgs
